cfep-05.md

```diff
@@ -0,0 +1,136 @@
+anaconda.org so that the more adventurous users can try our these pre-release
+packages.
+
+## Motivation
+
+Pypi allows package maintainers to upload "prerelease" builds which users can
+then install via
+
+```
+pip install PACKAGE_NAME --pre
+```
+
+The same semantics are slightly more cumbersome within the `conda` ecosystem but
+are still manageable.  The analogous command with `conda` is (for example):
+
+```
+conda install -c conda-forge/label/rc_matplotlib matplotlib
+```
+
+We, as the conda-forge community need to arrive at a consensus for what labels
+we will affix to packages so that there is a consistent user experience.
+
+## Backwards Compatibility
+
+There are only minimal concerns of backwards compatibility.  There are a few
+relevant labels that package maintainers have affixed to their pre-release
+packages. These are "pre" (1), "prerelease" (2), "rc" (2) and "test" (1).  The
+numbers in parentheses represent the number of projects which have released
+these pre-release packages.  With six total projects using non-main labels, I do
+not think we have a serious concern with backwards compatibility due to changes
+that come about via this proposal.
+
+## Specification
+
+Mark the pre-release package on anaconda.org as "dev" or "rc". The semantics of
+these labels should generally follow the
+[guidelines](https://docs.python.org/devguide/devcycle.html#stages) that Python
+itself follows.
+
+- "rc": [Beta](https://docs.python.org/devguide/devcycle.html#beta) and [Release
+  Candidate](https://docs.python.org/devguide/devcycle.html#release-candidate-rc)
+  (RC). No new features. Bugfix only.
+
+- "dev": [Pre-Alpha](https://docs.python.org/devguide/devcycle.html#pre-alpha)
+  and [Alpha](https://docs.python.org/devguide/devcycle.html#alpha). These are
+  still packages that could see substantial changes
+  between the dev version and the final release.
+
+If you wish to add numbers to your "dev" or "rc" you should follow the
+[guidelines](http://conda.pydata.org/docs/spec.html#build-version-spec) put
+forth by Continuum regarding version sorting in `conda`. Also see the [source
+code for conda
+4.2.13](https://github.com/conda/conda/blob/4.2.13/conda/version.py#L93-L119).
+The tl;dr here is that conda sorts as follows:
+
+```
+< 1.0
+< 1.1dev1    # special case 'dev'
+< 1.1.0dev1  # special case 'dev'
+== 1.1.dev1   # 0 is inserted before string
+< 1.1.0rc1
+< 1.1.0
+```
+
+So make sure that you ***tag*** your package in such a way that the package name
+that conda-build spits out will sort the package uploaded with an "rc" label
+higher than the package uploaded with the "dev" label.
+
+To create a `dev` or `rc` package a PR can be issued into the `dev` or `rc` branch of the
+feedstock.
+This branch must change the `conda_forge_config.yaml` to point to the `dev_<package_name>` or `rc_<package_name>`.
+
+For example, matplotlib rc releases would include:
+```yaml
+...
+channels:
+  targets:
+    - [conda-forge, rc_matplotlib]
```

Comment:
This syntax is wrong.

Reply:
What is the correct syntax? I modified the existing matplotlib development branch https://github.com/conda-forge/matplotlib-feedstock/blob/devel/conda-forge.yml

Reply:
See point 4 at https://github.com/conda-forge/conda-smithy#making-a-new-feedstock

There are 2 files.
1. `recipe/conda_build_config.yaml` - this is what actually works
2. `conda-forge.yml` - this was the conda-smithy <3 way, but this was partially removed. This is only used for the README now. Ideally someone should fix the README rendering to use `recipe/conda_build_config.yaml`

Reply:
Thank you!